src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/CancellationTokenParametersMustComeLast.cs

```diff
@@ -63,6 +63,11 @@ public override void Initialize(AnalysisContext context)
                         return;
                     }
 
+                    if (!symbolContext.Options.MatchesConfiguredVisibility(Rule, methodSymbol, symbolContext.Compilation, symbolContext.CancellationToken))
```

Comment:
I think we should not alter default semantics of this analyzer - it should flag all APIs by default, but end users can configure it to only configure specific API surfaces.
```suggestion
                    if (!symbolContext.Options.MatchesConfiguredVisibility(Rule, methodSymbol, symbolContext.Compilation, symbolContext.CancellationToken, defaultRequiredVisibility: SymbolVisibilityGroup.All))
```